WalletWasabi.Fluent/Behaviors/DialogTransitionAttachedBehavior.cs

```diff
@@ -72,7 +72,7 @@ private static void AnimateImplicit(Control control, TimeSpan opacityDuration, b
 
 		var scaleAnimation = compositor.CreateVector3KeyFrameAnimation();
 		scaleAnimation.Target = "Scale";
-		if (enableScale)
+		if (enableScale && false) // TODO: Remove when Avalonia fixes
 		{
 			scaleAnimation.InsertExpressionKeyFrame(0f, "Vector3(0.96+(1.0-0.96)*this.Target.Opacity, 0.96+(1.0-0.96)*this.Target.Opacity, 0)", fluentEasing);
```

Comment:
@wieslawsoltes This seems to be a bug in the current build of Avalonia, and I didn't see it reported. The Zooming animation doesn't work, `this.Target.Opacity` seems to have a bug because when set nothing is rendered, when I remove it everything works fine. Can you please confirm that it is a bug and open an issue on Avalonia side? Thanks!

Reply:
I opened an issue on Avalonia side: https://github.com/AvaloniaUI/Avalonia/issues/16774